src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintExtension.kt

```diff
@@ -16,4 +16,14 @@ open class KtlintExtension {
      * Enable debug mode.
      */
     var debug = false
+
+    /**
+     * Report output format.
+     *
+     * Possible values are: plain, checkstyle, json.
```

Comment:
Only one at a time.
 
Enum may be not a bad idea, but harder to maintain - like adding a new one and check supported version.